server/src/core/stat/types.ts

```diff
@@ -0,0 +1,36 @@
+import { UserService } from '../user';
+import { TransactionService } from '../payment'
+
+export interface StatsArgs {
+  users: UserService;
+  transactions: TransactionService
+}
+
+export interface Stat {
+  _id: string,
+  numContributors: number,
+  totalContributed: number,
+  numBeneficiaries: number, 
+  totalDistributed: number,
+  createdAt: Date,
```

Comment:
I don't think it's necessary to have a `createdAt` field since this is a singleton. Let's just have an `updatedAt` field.